theme/src/components/layout.js

```diff
@@ -24,7 +24,7 @@ function Layout({children, pageContext}) {
     <Flex flexDirection="column" minHeight="100vh">
       <Head title={title} description={description} />
       <Header />
-      <Flex flex="1 1 auto" flexDirection="row">
+      <Flex flex="1 1 auto" flexDirection="row" css={{'zIndex': '0'}}>
```

Comment:
[Minor] I don't think these need to be in quotes :)

```suggestion
      <Flex flex="1 1 auto" flexDirection="row" css={{zIndex: 0}}>
```